features/step_definitions/basic_steps.rb

```diff
@@ -31,14 +31,14 @@ def page_path_from(page_name)
     when 'thaitanic' then restaurant_path(Restaurant.find_by(name: page_name))
     when 'landing' then root_path
     when 'sign up' then new_user_registration_path
-    when 'cart' then charges_path()
+    when 'checkout' then charges_path()
   end
 end
 
 Then("I should be on the {string} page") do |page|
   sleep(5)
   case page
```

Comment:
Why are we using a case statement if we only have a single case? I'd re-use the `page_path_from` here to get the path to test for.

Reply:
Will fix this :)